packages/rum-recorder/src/domain/record/types.ts

```diff
@@ -216,8 +219,8 @@ export interface ViewportResizeDimention {
 export type ViewportResizeCallback = (d: ViewportResizeDimention) => void
 
 export interface InputValue {
-  text: string
-  isChecked: boolean
+  text?: string
+  isChecked?: boolean
```

Comment:
Could it be  `{ text: string } | { isChecked: boolean }`?

Reply:
I initially did that but it complexify the comparison here: https://github.com/DataDog/browser-sdk/blob/6c894daf216141129ed9008464ae37af5d318246/packages/rum-recorder/src/domain/record/observer.ts#L198 (we'd have to do a bunch of cast, or check that each property is set before accessing it). Since I don't think that it matters too much I chose to keep it simple, but if you prefer I can change that.

Reply:
I'd be in favor of doing it, it would clarify that it is one or the other